formidable/forms/__init__.py

```diff
@@ -113,10 +114,34 @@ class MyForm(FormidableForm):
     """
 
     @classmethod
-    def to_formidable(cls, label, description=u''):
-        form = Formidable.objects.create(label=label, description=description)
+    def to_formidable(cls, label=None, description=None, instance=None):
+        if not instance:
+            if not label:
+                raise ValueError("On creation mode, label value is mandatory.")
```

Comment:
Agree with the error message. But, I saw things like "You are passing a value in your method, this value is wrong (because None, and we can't). So ValueError is raised"

Why KeyError ?